labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>That's great! This section will certainly ring a bell for NLP practitioners (which we have many).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>:)</p>